components/display/include/display.hpp

```diff
@@ -269,23 +265,20 @@ class Display : public BaseComponent {
    * @param rotation Default / initial rotation of the display.
    * @param task_config Configuration for the task that runs the lvgl tick
    */
-  void init(flush_fn flush_callback, bool sw_rotation_enabled, Rotation rotation,
+  void init(lv_display_flush_cb_t flush_callback, bool sw_rotation_enabled, Rotation rotation,
             const Task::BaseConfig &task_config) {
     lv_init();
 
+    display_ = lv_display_create(width_, height_);
+
     // Configure the LVGL display buffer with our pixel buffers
-    lv_disp_draw_buf_init(&disp_buffer_, vram_0_, vram_1_, display_buffer_px_size_);
+    lv_display_set_buffers(display_, vram_0_, vram_1_, vram_size_bytes(),
+                           LV_DISPLAY_RENDER_MODE_PARTIAL);
 
-    lv_disp_drv_init(&disp_driver_);
-    disp_driver_.draw_buf = &disp_buffer_;
-    disp_driver_.flush_cb = flush_callback;
-    disp_driver_.sw_rotate = (uint8_t)sw_rotation_enabled;
```

Comment:
Is this not a configuration that can be used anymore? I'm noticing that some of the examples aren't rotated correctly anymore.